docs/extensibility.md

```diff
@@ -120,38 +120,64 @@ wp.hooks.addFilter(
 
 _Note:_ This filter must always be run on every page load, and not in your browser's developer tools console. Otherwise, a [block validation](https://wordpress.org/gutenberg/handbook/block-api/block-edit-save/#validation) error will occur the next time the post is edited. This is due to the fact that block validation occurs by verifying that the saved output matches what is stored in the post's content during editor initialization. So, if this filter does not exist when the editor loads, the block will be marked as invalid.
 
-## Adding a sidebar
-
-**Warning:** This is an experimental API, and is subject to change or even removal.
-
-### Registering a sidebar
+## Extending the editor's UI (Slot and Fill)
 
-`wp.editPost.__experimentalRegisterSidebar( name: string, settings: { title: string, render: function } )`
+Extending the editor's UI is done with in `registerPlugin` API, and allows you to define all your plugin's UI elements in one place.
 
-This method takes a sidebar `name` and a `settings` object, containing a title and a render function. The name should contain a namespace prefix (Example: my-plugin/my-custom-sidebar).
+`wp.editPost.registerPlugin( { name: string, render: function } )`
 
+This method takes one argument: 
+- An object containing the following data:
+  - `name`: A string identifying the plugin. Must be unique across all registered plugins.
+  - `render`: A component containing the UI elements to be rendered. See the list below for all available UI elements.
+  
 **Example**
 
 ```js
-wp.editPost.__experimentalRegisterSidebar( 'my-plugin/my-custom-sidebar', {
-	render: function mySidebar() {
-		return <p>This is an example</p>;
-	},
+const Fragment = wp.element.Fragment;
+const PluginSidebar = wp.editPost.PluginSidebar;
+
+const Component = () => (
+	<Fragment>
+        <PluginSidebar name="first-sidebar-name" title="My Sidebar">
+            <h1>Content of the first sidebar</h1>
+        </PluginSidebar>
+        <PluginSidebar name="second-sidebar-name" title="My Second Sidebar">
+            <h1>Content of the second sidebar</h1>
+        </PluginSidebar>
+	</Fragment>
+);
+
+wp.editPost.registerPlugin( {
+    name: 'plugin-name',
+    render: Component,
 } );
 ```
 
-### Activating a sidebar
+You can activate the sidebars using the following lines:
 
-`wp.editPost.__experimentalActivateSidebar( name: string )`
+`wp.data.dispatch( 'core/edit-post' ).openGeneralSidebar( 'plugin-name/first-sidebar-name' );`
+`wp.data.dispatch( 'core/edit-post' ).openGeneralSidebar( 'plugin-name/second-sidebar-name' );`
```

Comment:
![screen shot 2018-03-08 at 15 08 44](https://user-images.githubusercontent.com/699132/37155227-a31a8b22-22e2-11e8-8786-ee9c22a19e59.png)
Not sure why, but they display in one line...
It might help to treat it as a code snippet.